[2022-07-19 11:28:12,428] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:27:00+00:00 [queued]>
[2022-07-19 11:28:12,457] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:27:00+00:00 [queued]>
[2022-07-19 11:28:12,458] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:28:12,459] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 11:28:12,460] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:28:12,499] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 11:27:00+00:00
[2022-07-19 11:28:12,508] {standard_task_runner.py:52} INFO - Started process 11242 to run task
[2022-07-19 11:28:12,516] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T11:27:00+00:00', '--job-id', '583', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpsnqkhhnz', '--error-file', '/tmp/tmpu7deb4uf']
[2022-07-19 11:28:12,519] {standard_task_runner.py:80} INFO - Job 583: Subtask respond
[2022-07-19 11:28:12,674] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:27:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 11:28:12,922] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T11:27:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T11:27:00+00:00
[2022-07-19 11:28:22,126] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 11:28:22,135] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 11:28:22,192] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T112700, start_date=20220719T112812, end_date=20220719T112822
[2022-07-19 11:28:22,258] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 11:28:22,313] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
